__tests__/api/id/form/apiUsers.test.ts

```diff
@@ -5,21 +5,19 @@
 /* eslint-disable  @typescript-eslint/no-explicit-any */
 
 import { createMocks, RequestMethod } from "node-mocks-http";
-import { unstable_getServerSession } from "next-auth/next";
+import { getServerSession } from "next-auth/next";
 import apiUsers from "@pages/api/id/[form]/apiusers";
-import { logAdminActivity } from "@lib/adminLogs";
 import { prismaMock } from "@jestUtils";
 import { Prisma } from "@prisma/client";
 import { Session } from "next-auth";
 import { Base, ManageForms, getUserPrivileges } from "__utils__/permissions";
 
 jest.mock("next-auth/next");
-jest.mock("@lib/adminLogs");
 
 //Needed in the typescript version of the test so types are inferred correclty
-const mockGetSession = jest.mocked(unstable_getServerSession, { shallow: true });
+const mockGetSession = jest.mocked(getServerSession, { shallow: true });
 
-describe("/id/[forms]/owners", () => {
+describe.skip("/id/[forms]/owners", () => {
```

Comment:
This skip is intentional? Suspect it is but double checking

Reply:
It is because we've disabled the API endpoint by passing in [] to `allowedMethods`.

Reply:
Right, I remember seeing that for a POST endpoint but not others. Need more coffee. Good - will resolve this :)